### PR TITLE
[webui] Improve trial parameter for retiarii experiment

### PR DESCRIPTION
**Pull request by Lijiaoa**
### Description ###
for retiarii experiment:
- show trial normal parameter
  
image

- show real trial parameter by clicking `Original parameters` such as this picture:

image

### Checklist ###
  - [ ] test case
  - [ ] doc
  - [x] these PR should be merged after the PR: https://github.com/microsoft/nni/pull/4304

### How to test ###

1. run example `examples/nas/multi-trial/mnist`, 
- [ ] see the hyper-parameter graph
- [x] see trial parameter in the table (oview && detail table)
- [x] copy trial parameters
- [x] confirm trail `Compare` model in the detail page
- [ ] confirm `Default metric | Intermediate result` graph's tooltip

2. [x] run nested example and see the website
3. [x] run mnist example and see the website

### CODE REVIEW

**Review by QuanluZhang — APPROVED** [review]

**Comment by liuzhe-lz** [comment]
Merge after #4304

File: ts/webui/src/components/public-child/OpenRow.tsx [hunk 1]
```diff
@@ -66,10 +78,12 @@ class OpenRow extends React.Component<OpenRowProps, OpenRowState> {
     };
 
     render(): React.ReactNode {
-        const { isHidenInfo, typeInfo, info } = this.state;
+        const { isHidenInfo, typeInfo, info, showRetiaParamPanel } = this.state;
         const trialId = this.props.trialId;
         const trial = TRIALS.getTrial(trialId);
         const logPathRow = trial.info.logPath || "This trial's log path is not available.";
+        const originParameters = JSON.parse(JSON.stringify(trial.description.parameters));
```
Comment (liuzhe-lz) [inline ts/webui/src/components/public-child/OpenRow.tsx:85]: Seems no need to copy.

File: ts/webui/src/components/public-child/OpenRow.tsx [hunk 2]
```diff
@@ -66,10 +78,12 @@ class OpenRow extends React.Component<OpenRowProps, OpenRowState> {
...
+        const isHasVisualHyperParams = '_visual_hyper_params_' in originParameters;
```
Comment (liuzhe-lz) [inline ts/webui/src/components/public-child/OpenRow.tsx:86]: `hasVisualHyperParams`

File: ts/webui/src/components/public-child/OpenRow.tsx [hunk 3]
```diff
@@ -91,8 +105,21 @@ class OpenRow extends React.Component<OpenRowProps, OpenRowState> {
                                             text='Copy as json'
                                             styles={{ root: { width: 128, marginRight: 10 } }}
                                         />
+                                        {isHasVisualHyperParams && (
+                                            <DefaultButton
+                                                onClick={this.isshowRetiaParamPanel}
+                                                text='Origin parameters'
```
Comment (liuzhe-lz) [inline ts/webui/src/components/public-child/OpenRow.tsx:111]: Original parameters

**Review by liuzhe-lz — APPROVED** [review]

File: ts/webui/src/components/public-child/OpenRow.tsx (hunk 2 above)
Comment (Lijiaoa) [reply ts/webui/src/components/public-child/OpenRow.tsx]: had fixed

File: ts/webui/src/components/public-child/OpenRow.tsx (hunk 3 above)
Comment (Lijiaoa) [reply ts/webui/src/components/public-child/OpenRow.tsx]: had fixed

File: ts/webui/src/components/public-child/OpenRow.tsx (hunk 1 above)
Comment (Lijiaoa) [reply ts/webui/src/components/public-child/OpenRow.tsx]: had fixed

**Comment by Lijiaoa** [comment]
latest commit is for this:

image

File: ts/webui/src/components/public-child/OpenRow.tsx [hunk 4]
```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
 import * as copy from 'copy-to-clipboard';
-import { Stack, PrimaryButton, Pivot, PivotItem } from '@fluentui/react';
+import { Stack, PrimaryButton, Pivot, PivotItem, DefaultButton } from '@fluentui/react';
 import { Trial } from '../../static/model/trial';
-import { MANAGER_IP } from '../../static/const';
+import { MANAGER_IP, RETIARIIPARAMETERS } from '../../static/const';
 import { EXPERIMENT, TRIALS } from '../../static/datamodel';
+import { filterParameter } from '../../static/function';
```
Comment (ultmaster) [inline ts/webui/src/components/public-child/OpenRow.tsx:7]: reformatParameter

**Comment by Lijiaoa** [comment]
- [ ] test

**Pull request merged**